lib/internal/test_runner/runner.js

```diff
@@ -536,6 +543,10 @@ function run(options = kEmptyObject) {
     setup,
     only,
     globPatterns,
+    coverage,
+    lineCoverage,
+    branchCoverage,
+    functionCoverage,
```

Comment:
```suggestion
    lineCoverage = 0,
    branchCoverage = 0,
    functionCoverage = 0,
```

Wondering if we really need to throw on these if `!coverage`? Could we just silently ignore them?